examples/oci-image-verification/go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/sigstore/sigstore-go/examples/oci-image-verification
 
-go 1.23.0
-
-toolchain go1.24.0
+go 1.24.1
```

Comment:
Unless we need a feature from 1.24, let's keep it at 1.23. When you build the example, the version of the Go toolchain that you build with is what is used.